web/src/beta/lib/core/engines/Cesium/Feature/Polyline/index.tsx

```diff
@@ -50,7 +52,18 @@ export default function Polyline({ id, isVisible, property, geometry, layer, fea
     [coordinates ?? []],
     isEqual,
   );
-  const material = useMemo(() => toColor(strokeColor), [strokeColor]);
+
+  const { viewer } = useCesium();
+  const entity = viewer ? findEntity(viewer, layer?.id, feature?.id) : undefined;
```

Comment:
I think you can use the `ref` from [EntityExt](https://github.com/reearth/reearth/blob/a9d933fdd8782facef7393dac4a6522bc980360d/web/src/beta/lib/core/engines/Cesium/Feature/Polyline/index.tsx#L79) to get the entity without searching all entities.

Reply:
Updated with ref. Thank you!